src/screens/ContactScreen/elements/ItemUser.js

```diff
@@ -33,7 +34,7 @@ const ItemUser = ({photo, username, followed, onPress, userid}) => {
         style={[
           styles.containerCard,
           {
-            backgroundColor: isSelect ? '#00ADB526' : 'white'
+            backgroundColor: isSelect ? COLORS.holytosca15percent : COLORS.white
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Good job on replacing the hardcoded color values with constants from a theme file. This will make it easier to manage and update the color scheme of the application in the future.
```diff
- backgroundColor: isSelect ? '#00ADB526' : 'white'
+ backgroundColor: isSelect ? COLORS.holytosca15percent : COLORS.white
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->